docs/elements/session.html

```diff
@@ -0,0 +1,213 @@
+        php="
+    /**
+     * @Given the session flashes the following data:
+     */
+    public function theSessionFlashesTheFollowingData(TableNode $table): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>theSessionShouldFlashTheFollowingData</code> method verifies that the session contains the specified
+        flash data. This is useful for asserting that the session flash data has the expected values.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the session flashes the following data:
+        | key       | value                |
+        | message   | Operation successful |
+        | alertType | success              |"
+        php="
+    /**
+     * @Then the session should flash the following data:
+     */
+    public function theSessionShouldFlashTheFollowingData(TableNode $table): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>theSessionShouldHaveKey</code> method verifies that the session contains a specified key. This is
+        useful for asserting the presence of specific session keys.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the session should have key 'user_id'"
+        php="
+    /**
+     * @Then the session should have key :key
+     */
+    public function theSessionShouldHaveKey(string $key): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>theSessionShouldNotHaveKey</code> method verifies that the session does not contain a specified key.
+        This is useful for asserting the absence of specific session keys.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the session should not have key 'user_id'"
+        php="
+    /**
+     * @Then the session should not have key :key
+     */
+    public function theSessionShouldNotHaveKey(string $key): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>theSessionShouldHaveKeyWithValue</code> method verifies that a session key contains a specified value.
+        This is useful for asserting that session keys have the expected values.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Then the session should have key 'user_role' with value 'admin'"
+        php="
+    /**
+     * @Then the session should have key :key with value :value
+     */
+    public function theSessionShouldHaveKeyWithValue(string $key, string $value): void"></code-snippet>
+
+<article-paragraphs>
+    <p>
+        The <code>theSessionForgetsKey</code> method removes a specified key from the session. This is useful for
+        scenarios where specific session keys need to be cleared.
+    </p>
+</article-paragraphs>
+
+<code-snippet
+        gherkin="Given the session forgets the key 'user_id'"
+        php="
+    /**
+     * @Given the session forgets the key :key
+     */
+    public function theSessionForgetsKey(string $key): void"></code-snippet>
+    
```

Comment:
Here GitHub shows that there is no empty line at the end of the file.

![image](https://github.com/blumilksoftware/blt/assets/56546832/cbc2c61f-bdf2-4868-aff5-b3cae31e7b41)

Reply:
![6b822d0e4bab6a0bbc42da32f28bdcf2](https://github.com/blumilksoftware/blt/assets/115737114/abb091c9-8d11-414f-ba60-ffbf5e1e7abb)

213 is empty